s3.go

```diff
@@ -94,14 +95,12 @@ retry:
 	attempt++
 	obj, err := s.client.GetObject(s.bucket, name, minio.GetObjectOptions{})
 	if err != nil {
-		if attempt < s.opt.ErrorRetry {
-			goto retry
-		}
 		return nil, errors.Wrap(err, s.String())
```

Comment:
I believe the retry here is still necessary, since it's still going to make a network call to lookup the object. That initial call could fail as well

Reply:
According to code `GetObject()` return error only here https://github.com/minio/minio-go/blob/master/api-get-object.go#L37 and here https://github.com/minio/minio-go/blob/master/api-get-object.go#L40. But not I think this behavior can change I better bring back this lines.